app/scripts/modules/amazon/src/serverGroup/configure/wizard/capacity/CapacitySelector.tsx

```diff
@@ -55,11 +55,10 @@ export class CapacitySelector extends React.Component<ICapacitySelectorProps> {
     this.setState({});
   };
 
-  private capacityFieldChanged = (fieldName: 'min' | 'max' | 'desired', value: string) => {
+  private capacityFieldChanged = (fieldName: 'min' | 'max' | 'desired', value: number | string) => {
     const { command, setFieldValue } = this.props;
-    const num = Number.parseInt(value, 10);
     command.capacity = { ...command.capacity };
-    command.capacity[fieldName] = num;
+    command.capacity[fieldName] = value;
```

Comment:
How do you feel about doing `isNaN(value) ? value : Number.parseInt(value, 10)` here? Too much?

Reply:
That's actually what I originally had, but the union type `number | string` screws that up. It would actually have to be something like:
```
if (typeof value === 'string') {
  const parsed = parseInt(value, 10);
  command.capacity[fieldName] = isNaN(parsed) ? value : parsed;
} else {
  command.capacity[fieldName] = value;
}
```
Which feels like a bit much here, but let me try pushing it down and making the `SpelNumberInput` responsible for all the parsing. I think that feels ok.